### PR TITLE
Address #23: latitudinal binning and variable latitudinal range

### LINKED ISSUE #23: lat_bins() occurrence binning and variable latitudinal range
1. Currently, if occurrences fall on boundaries in lat_bins() they are assigned to the higher bin number. It could be useful to add functionality to allow occurrences to be binned into both bins.
2. Currently, lat_bins() only covers the complete latitudinal range (i.e. -90 to 90 latitude). It could be useful to add functionality to allow user input ranges (e.g. -60 to 60).

### PR DESCRIPTION
**Pull request by LewisAJones**
These updates enable users to:
- `lat_bins`: input a user-defined latitudinal range.
- `bin_lat`: bin occurrences into both latitudinal bins if the occurrence falls on the bin boundary. It does so by duplicating the occurrence (row).

Fixes #23.

### CODE REVIEW

**Review by SofiaGalv — COMMENTED** [review]
Everything looks fine to me. I think allowing the user to define the bins is great!

File: R/bin_lat.R [hunk 1]
```diff
@@ -12,6 +12,11 @@
 #' @param lat \code{character}. The name of the column you wish to be treated
 #' as the input latitude (e.g. "lat" or "p_lat"). This column should contain
 #' numerical values. Defaults to "lat".
+#' @param boundary \code{logical}. If \code{TRUE}, occurrences
+#' falling on the boundaries of latitudinal bins will be duplicated and
+#' binded to the dataframe after being assigned to both bins.
```
Comment (bethany-j-allen) [inline R/bin_lat.R:17]:
```suggestion
#' assigned to both bins.
```

**Review by bethany-j-allen — APPROVED** [review]
Looks great Lewis! I've made one grammatical suggestion but otherwise I think this is good to go.

**Comment by LewisAJones** [comment]
Thanks both! You turned this around incredibly quickly. As these are not new functions and only expanding current functionality we can probably even get this into the next version :-)

**Pull request merged**